bin/dockerfile_test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+function help() {
+  echo "Builds the docker image and runs the tests against the development enviroment.
+
+  Run in an authenticated shell with permission to access ssm params in
+  forms-deploy using the gds-cli or aws-vault
+
+  Usage: $0
+
+  Example:
+  gds-cli aws forms-deploy-readonly -- $0
+  "
+exit 1
+}
+
+if [[ "$1" == "help" ]]; then
+  help
+fi
+
+if [[ -z "$AWS_ACCESS_KEY_ID" ]]; then
+  echo "No AWS credentials found"
+  help
+fi
+
+echo 'Building image'
+image_tag="test_e2e_$(date +%s)"
```

Comment:
```suggestion
image_tag="test_e2e_$(date +'%Y-%m-%d_%H-%M')"
```
For local builds I'd find it more useful to have a date/time I can read

Reply:
That's good suggestion and I'll incorporate it, thanks Cat 👍